### ISSUE TITLE
Error with empty files

### ISSUE THREAD

**Issue by tribut** [thumbsup/thumbsup#73]
## Bug report

```
Thumbsup version: 2.1.0
Node version: 6.11.4
NPM version: 3.10.10
Operating system: Debian Stretch
```

When building a site that contains an empty file, thumbsup throws an error and does not generate the gallery. The error message does not help identify the problem. To reproduce:

~~~~
$ mkdir in out
$ touch in/empty.jpg
$ thumbsup --input in --output out

Finding media       [=========================] 1/1 done
Updating database   [=========================] 1/1 done
Processing photos   [=========================] up to date
Processing videos   [=========================] up to date
Creating albums     [=========================] done
Building website    [-------------------------] path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.relative (path.js:1234:5)
    at Media.<anonymous> (/home/www-bilder/.local/lib/node_modules/thumbsup/src/output-website/template.js:102:17)
    at eval (eval at createFunctionContext (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:6:74)
    at Object.prog [as inverse] (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at Media.<anonymous> (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:17:22)
    at eval (eval at createFunctionContext (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:5:35)
    at prog (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at execIteration (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
    at Object.<anonymous> (/home/www-bilder/.local/lib/node_modules/thumbsup/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:61:13)
~~~~

**Comment by eephyne** [comment]
same as https://github.com/thumbsup/thumbsup/issues/78

**Comment by rprieto** [comment]
Thanks for reporting this, it will be fixed in the upcoming `2.2.0` release.
